src/main/java/io/moderne/connect/utils/GitLabYaml.java

```diff
@@ -77,6 +78,7 @@ Map<String, Object> prepareYamlMap() {
     @Value
     @Builder
     public static class Job {
+        String image;
         Cache cache;
         Stage stage;
         @Singular
```

Comment:
Quick readability request that came up in our standup yesterday: a new line before annotated fields.
```suggestion

        @Singular
```

Reply:
Done!